scripts/build_perf.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
This introduces a bash dependency in u-root, and not all GOOS choices have bash (e.g. Plan 9). It would be nice if you could redo this in Go.